flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumerBaseMigrationTest.java

```diff
@@ -275,20 +274,18 @@ public void testRestoreFromEmptyStateWithPartitions() throws Exception {
         }
 
         // assert that there are partitions and is identical to expected list
-        assertTrue(consumerFunction.getSubscribedPartitionsToStartOffsets() != null);
-        assertTrue(!consumerFunction.getSubscribedPartitionsToStartOffsets().isEmpty());
-        assertEquals(
-                expectedSubscribedPartitionsWithStartOffsets,
-                consumerFunction.getSubscribedPartitionsToStartOffsets());
+        assertThat(consumerFunction.getSubscribedPartitionsToStartOffsets()).isNotNull();
+        assertThat(!consumerFunction.getSubscribedPartitionsToStartOffsets().isEmpty()).isTrue();
+        assertThat(consumerFunction.getSubscribedPartitionsToStartOffsets())
+                .isEqualTo(expectedSubscribedPartitionsWithStartOffsets);
```

Comment:
Under the hood `isNotEmpty()`, `isEmpty()` check for is not null, also could be joined
```suggestion
        assertThat(consumerFunction.getSubscribedPartitionsToStartOffsets())
                .isNotEmpty()
                .isEqualTo(expectedSubscribedPartitionsWithStartOffsets);
```